peps/pep-0750.rst

```diff
@@ -53,638 +34,500 @@ deferred pending further experience with f-strings. Work on this PEP was
-  function. Example: ``'some-formatspec'``.
+.. code-block:: python
 
-In all cases, the tag function determines what to do with valid ``Interpolation``
-attributes.
+    trade = 'shrubberies'
+    t = rt'Did you say "{trade}"?\n'
+    assert t.args[0] == r'Did you say "'
+    assert t.args[2] == r'"?\n'
 
-In the CPython reference implementation, implementing ``Interpolation`` in C would
-use the equivalent `Struct Sequence Objects
-<https://docs.python.org/3/c-api/tuple.html#struct-sequence-objects>`_ (see
-such code as `os.stat_result
-<https://docs.python.org/3/library/os.html#os.stat_result>`_). For purposes of this
-PEP, here is an example of a pure Python implementation:
+In this example, the ``\n`` is treated as two separate characters 
+(a backslash followed by 'n') rather than a newline character. This is 
+consistent with Python's raw string behavior.
 
-.. code-block:: python
+As with regular template strings, interpolations in raw template strings are 
+processed normally, allowing for the combination of raw string behavior and 
+dynamic content.
 
-    class InterpolationConcrete(NamedTuple):
-        getvalue: Callable[[], Any]
-        expr: str
-        conv: Literal['a', 'r', 's'] | None = None
-        format_spec: str | None = None
 
 Interpolation Expression Evaluation
 -----------------------------------
 
-Expression evaluation for interpolations is the same as in :pep:`498#expression-evaluation`,
-except that all expressions are always implicitly wrapped with a ``lambda``:
+Expression evaluation for interpolations is the same as in :pep:`498#expression-evaluation`:
 
     The expressions that are extracted from the string are evaluated in the context
-    where the tag string appeared. This means the expression has full access to its
+    where the template string appeared. This means the expression has full access to its
     lexical scope, including local and global variables. Any valid Python expression
     can be used, including function and method calls.
 
-However, there's one additional nuance to consider, `function scope
-<https://docs.python.org/3/reference/executionmodel.html#resolution-of-names>`_
-versus `annotation scope
-<https://docs.python.org/3/reference/executionmodel.html#annotation-scopes>`_.
-Consider this somewhat contrived example to configure captions:
+Template strings are evaluated eagerly from left to right, just like f-strings. This means that
+interpolations are evaluated immediately when the template string is processed, not deferred
+or wrapped in lambdas.
 
-.. code-block:: python
-
-    class CaptionConfig:
-        tag = 'b'
-        figure = f'<{tag}>Figure</{tag}>'
 
-Let's now attempt to rewrite the above example to use tag strings:
-
-.. code-block:: python
+Exceptions
+----------
 
-    class CaptionConfig:
-        tag = 'b'
-        figure = html'<{tag}>Figure</{tag}>'
+Exceptions raised in t-string literals are essentially the same as those raised in f-strings.
```

Comment:
"are essentially" seems waffly to me. Are they the same? If not, how so? I suggest removing "essentially."

Reply:
Agree. Fixed.